python-libraries/data-platform-catalogue/data_platform_catalogue/entities.py

```diff
@@ -1,200 +1,265 @@
-from dataclasses import dataclass, field
 from datetime import datetime
-from enum import Enum, auto
-from typing import Any
+from enum import Enum
+from typing import Optional
+
+from pydantic import BaseModel, Field
 
 DATAHUB_DATE_FORMAT = "%Y%m%d"
 
 
-@dataclass
-class CatalogueMetadata:
-    name: str
-    description: str
-    owner: str
-    tags: list[str] = field(default_factory=list)
+class DatabaseStatus(Enum):
+    PROD = "PROD"
+    PREPROD = "PREPROD"
+    DEV = "DEV"
+
 
+class RelationshipType(Enum):
+    PARENT = "PARENT"
+    PLATFORM = "PLATFORM"
 
-@dataclass
-class DataLocation:
+
+# Needs refining
```

Comment:
Question: Do we use this classification anywhere at the moment?

If no, let's drop this entirely. 

If yes, we need to add in the missing values, possibly rename it.

Reply:
Just a custom property. There was some talk of adding in subsets of official (which would be official sensitive too) see https://security-guidance.service.justice.gov.uk/government-classification-scheme/#government-classification-scheme

but this wasn't ever really agreed and not sure if it's useful content